cmd/pulumi-test-language/tests/testdata/l2-builtin-try-with-output/main.pp

```diff
@@ -0,0 +1,36 @@
+resource "component1" "component:index:ComponentCustomRefOutput" {
+  value = "foo-bar-baz"
+}
+
+# When accessing an output of a component inside a direct call to try we should have to use apply.
+#
+# TODO(pulumi/pulumi#18895) When value is directly a scope traversal inside the
+# output this fails to generate the "apply" call. eg if the output's internals
+# are `value = componentTried.value`
```

Comment:
Assuming we agree with my comments to move to `l1-builtin-try`, I'd skip these comments too. As a reader it's not clear to me -- do they affect this test, is this test guarding against them/incomplete in the face of them etc.?